### DISCUSSION TITLE
Errors on modals lost after upgrading to Laravel 9

### DISCUSSION THREAD

**Discussion by andcl** [livewire/livewire#4721] [1. Help]
Hello, 

I recently upgraded Laravel to v9, and the errors that were shown in a modal before the upgrade are now notified in a much less verbose error in JS console always saying:
```
Uncaught (in promise) SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at index.js:69:58
```

That portion of index.js is the following: 

```
...
.then(response => {
  if (response.ok) {
      response.text().then(response => {
          if (this.isOutputFromDump(response)) {
              this.onError(message)
              this.showHtmlModal(response)
          } else {
              this.onMessage(message, JSON.parse(response))  <--- This is the line that fails
          }
      })
} else {
...
isOutputFromDump(output) {
    return !!output.match(/<script>Sfdump\(".+"\)<\/script>/)
}
```

My livewire version: latest **v2.10.4**.
Investigating, I have figured out that error responses that before returned `true` in `this.isOutputFromDump(response)` function, are now returning `false` and that is why I am receiving errors in that way.

Could it be related to the update of the ignition error package in Laravel 9, which is returning errors differently from previous version and then Livewire does not classify them accordingly?

If that is not the case, then how could I see errors like before?

Thanks in advance.

**Comment by joshhanley** [comment]
@andcl ah yeah you could be right about the ignition change. Any chance you could create a small example repo which reproduces the issue with as little code as possible so we can investigate it?

Thanks!

**Comment by joshhanley** [comment]
@andcl I just did a test of this on Laravel 9 and it all seems to be working ok. Is this what you mean?

<img width="1840" alt="image" src="https://user-images.githubusercontent.com/882837/156123110-af97e54e-9fc7-4816-9c5d-64ac1370528b.png">

**Comment by iafilin** [comment]
I confirm the problem by running the application using Laravel Octane.
<img width="464" alt="image" src="https://user-images.githubusercontent.com/26049079/198894687-94f0853d-15fd-460a-9b08-c3926a0c1e4d.png">
<img width="462" alt="image" src="https://user-images.githubusercontent.com/26049079/198894701-f6c1f59c-3a13-4ead-8720-697eb43f0bf7.png">
<img width="442" alt="image" src="https://user-images.githubusercontent.com/26049079/198894718-7643753b-27e9-48e1-96b8-fa80b043dc33.png">

**Reply by joshhanley** [reply]
@iafilin thanks for sharing! Any chance you could provide some code and instructions to replicate the issues so we can have a look? Thanks!
